### PR TITLE
Update the docs for vectorio

### PR DESCRIPTION
**Pull request by lesamouraipourpre**

### CODE REVIEW

File: shared-bindings/vectorio/Circle.c [hunk 1]
```diff
@@ -81,6 +81,21 @@ const mp_obj_property_t vectorio_circle_radius_obj = {
...
+//|
+//|     y : int
+//|     """Y position of the center point of the circle in the parent."""
+//|
+//|     location : Tuple[int,int]
+//|     """(X,Y) position of the center point of the circle in the parent."""
+//|
+//|     pixel_shader : Union[displayio.ColorConverter,displayio.Palette]
+//|     """The pixel shader of the circle."""
+//|
```
Comment (microDev1) [inline shared-bindings/vectorio/Circle.c:97]: Why repeat these? Definition for these properties is already available in the parameters section.

**Review by microDev1 — COMMENTED** [review]
Thanks for the PR @lesamouraipourpre, just one comment regarding repetition.

File: shared-bindings/vectorio/Circle.c (hunk 1 above)
Comment (lesamouraipourpre) [reply shared-bindings/vectorio/Circle.c:97]: It makes it clearer that they are properties and be `get` and `set` after the object is constructed.
Before
Screenshot from 2021-09-12 12-56-23
After
Screenshot from 2021-09-12 13-01-04

File: shared-bindings/vectorio/Polygon.c [hunk 2]
```diff
@@ -86,6 +88,22 @@ const mp_obj_property_t vectorio_polygon_points_obj = {
...
+// Documentation for properties inherited from VectorShape.
+
+//|     x : int
+//|     """Initial screen X position of the 0,0 origin in the points list."""
+//|
+//|     y : int
+//|     """Initial screen Y position of the 0,0 origin in the points list."""
+//|
+//|     location : Tuple[int,int]
+//|     """Initial screen (X,Y) position of the 0,0 origin in the points list."""
```
Comment (microDev1) [inline shared-bindings/vectorio/Polygon.c:101]:
```suggestion
//|     """X position of the 0,0 origin in the points list."""
//|
//|     y : int
//|     """Y position of the 0,0 origin in the points list."""
//|
//|     location : Tuple[int,int]
//|     """(X,Y) position of the 0,0 origin in the points list."""
```

**Review by microDev1 — CHANGES_REQUESTED** [review]
LGTM! Just one more thing.

**Review by microDev1 — APPROVED** [review]

**Pull request merged**